src/App.tsx

```diff
@@ -8,7 +8,39 @@ import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
 
+import { Todo } from './types/Todo';
+import { getTodos } from './api';
+import { CompletedStatus } from './types/CompletedStatus';
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [isLoading, setIsLoading] = useState<boolean>(false);
+  const [query, setQuery] = useState<string>('');
+  const [selectedTodoId, setSelectedTodoId] = useState<number | null>(null);
+  const [filterStatus, setFilterStatus] = useState<CompletedStatus>(
+    CompletedStatus.all,
+  );
+
+  let flag = null;
```

Comment:
Consider changing `let flag = null;` to `const flag = null;` for better code clarity and to prevent unintended reassignment.